python/rand_forest.py

```diff
@@ -7,19 +7,41 @@
 import sys
 import time
 import pickle
-
+import logging
+import os
+import matplotlib.pyplot as plt
 import numpy as np
 import itertools
-
-from sklearn.cluster import KMeans
+from sklearn.cluster import Birch,KMeans
 from sklearn.ensemble import RandomForestClassifier
+from sklearn.cluster import DBSCAN,OPTICS, cluster_optics_dbscan
 
 import eqpy, ga_utils
 
-# list of ga_utils parameter objects
-transformer = None
 
+experiment_folder = os.getenv('TURBINE_OUTPUT')
+logging.basicConfig(format='%(message)s',filemode = 'a+',filename=os.path.join(experiment_folder,"iterations.log"),level=logging.DEBUG)
+
+with open(os.path.join(experiment_folder,'..','..',os.getenv('RAND_CONFIG_FILE'))) as f:
+    rand_config = json.load(f)
 
+clustering_method = rand_config['method']
+param1 = rand_config['param1'] #KMEANS->k, DBSCAN->eps, BIRCH->branching factor
+param2 = rand_config['param2'] #KMEANS->Nothing, DBSCAN->MinPts, BIRCH->threshold
+
+transformer = None
+fileDir = os.getcwd()
+previous_results_file = os.path.join(fileDir, '../../python/all_exps_DD.csv')
```

Comment:
Could we make this optional?
It is not necessarily the case that we will have such results beforehand.